### DISCUSSION TITLE
Mg-Graph remove role assignment

### DISCUSSION THREAD

**Discussion by ybar17** [microsoftgraph/msgraph-sdk-powershell#2849] [Q&A]
I am trying to remove the role assigned to user Active and Eligible.
Run this PS script to see if there are roles assigned to the user, 

`Connect-MgGraph -Scopes "RoleManagement.ReadWrite.Directory"

$UserID = "<User objectID>"
$user = Get-MgUser -UserId $UserID | select *`

In the return, I don't see that the user has any role assigned at all
see attached pic.
![2024-07-16 14_37_05-Administrator_ Windows PowerShell ISE](https://github.com/user-attachments/assets/d74ffdb1-b687-4759-8dfe-95b18a74786f)

**Comment by CupOfJoe-L** [comment]
If you are referring to Entra ID roles and want to see who is assigned what then I suggest you look into
`Get-MgRoleManagementDirectoryRoleAssignment`  referred to in [docs](https://learn.microsoft.com/en-us/graph/api/rbacapplication-list-roleassignments?view=graph-rest-1.0&tabs=http)
